apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -60,6 +73,15 @@ export function HeaderNav({ isIntercomOpened }: Props) {
   const segment = useSegment();
   const isSelfHosted = IS_DOCKER_HOSTED;
 
+  // Determine if the gizmo is visible
+  const [toggled, setToggled] = useState();
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Perhaps rename to something more semantic that doesn't require a comment.‏
```suggestion
  const [isGizmoVisible, setIsGizmoVisible] = useState();
```